features/support/base_urls.rb

```diff
@@ -2,6 +2,10 @@ def app_domain
   ENV["GOVUK_APP_DOMAIN"] || "integration.publishing.service.gov.uk"
 end
 
+def app_domain_internal
+  ENV["GOVUK_APP_DOMAIN_INTERNAL"] || ""
```

Comment:
I'm not sure if the `|| ""` is required. 

If it's not defined then it will return `nil`, and you could put that (`.nil?`) in the tertiary operator below.

Reply:
Updated